src/ClientSymbols/include/ClientSymbols/QSettingsBasedStorageManager.h

```diff
@@ -5,16 +5,34 @@
 #ifndef CLIENT_SYMBOLS_Q_SETTINGS_BASED_STORAGE_MANAGER_H_
 #define CLIENT_SYMBOLS_Q_SETTINGS_BASED_STORAGE_MANAGER_H_
 
+#include <QCoreApplication>
+#include <QSettings>
+#include <QString>
+#include <utility>
+
 #include "ClientSymbols/PersistentStorageManager.h"
 
 namespace orbit_client_symbols {
 
 class QSettingsBasedStorageManager : public PersistentStorageManager {
  public:
+  QSettingsBasedStorageManager()
+      : organization_(QCoreApplication::organizationName()),
+        application_(QCoreApplication::applicationName()) {}
+
+  QSettingsBasedStorageManager(QString organization, QString application)
+      : organization_(std::move(organization)), application_(std::move(application)) {}
+
   void SavePaths(absl::Span<const std::filesystem::path> paths) override;
   [[nodiscard]] std::vector<std::filesystem::path> LoadPaths() override;
   void SaveModuleSymbolFileMappings(const ModuleSymbolFileMappings& mappings) override;
   [[nodiscard]] ModuleSymbolFileMappings LoadModuleSymbolFileMappings() override;
+
+ private:
+  QSettings MakeSettings() { return QSettings{organization_, application_}; }
```

Comment:
I just looked at `https://doc.qt.io/qt-5/qsettings.html#accessing-settings-from-multiple-threads-or-processes-simultaneously` and it seems that when one QSettings saves something, this is visible to other QSettings objects immediately. So I think having a `QSettings settings_` private member should just work fine. I think that is a nicer solution than always calling MakeSettings